modules/manage/pages/tiered-storage.adoc

```diff
@@ -1,7 +1,7 @@
-= Tiered Storage
+= About Tiered Storage
```

Comment:
This also contains 'how-to' info. I suggest giving this a title like 'Enable Tiered Storage'.